### PR TITLE
remove retry strategy and circuit breaker set http timeout default to…

### PR DESCRIPTION
**Pull request by nataliataylor**
… 1 sec

### CODE REVIEW

File: src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs [hunk 1]
```diff
@@ -111,9 +111,9 @@ public int NumberOfFailuresBeforeTrippingCircuitBreaker
         public TimeSpan CircuitBreakerBreakDuration { get; set; } = new TimeSpan(0, 0, 0, seconds: 60);
 
         /// <summary>
-        ///     Http timeout.  Default is 30 seconds.
+        ///     Http timeout.  Default is 1 seconds.
         /// </summary>
-        public TimeSpan HttpTimeout { get; set; } = new TimeSpan(0, 0, 0, seconds: 30);
+        public TimeSpan HttpTimeout { get; set; } = new TimeSpan(0, 0, 0, seconds: 1);
```
Comment (kkanwar) [inline src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs:116]: Is there any way to make this user configurable?

**Review by kkanwar — APPROVED** [review]
Minor question, otherwise LGTM!

File: src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs (hunk 1 above)
Comment (nataliataylor) [reply src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs:116]: @kkanwar Do you want to add a second timeout option for authorization?

Comment (kkanwar) [reply src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs:116]: Not not at all, just wondering if the user can provide their chosen timeout value, and we use that for both auth and price opt requests.

Comment (nataliataylor) [reply src/Spresso.Sdk.Core/Auth/AuthTokenHandlerOptions.cs:116]: @kkanwar library already allows that. They create AuthTokenHandler with customized options and pass it to the PriceOptimizationsHandler during class instance creation.
`new PriceOptimizationsHandler(authTokenHandler, options);`

**Pull request merged**